x/claim/genesis.go

```diff
@@ -10,14 +10,22 @@ import (
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	// this line is used by starport scaffolding # genesis/module/init
 	k.SetParams(ctx, genState.Params)
+	err := k.SetClaimRecords(ctx, genState.ClaimRecords)
+	if err != nil {
+		panic(err) // if genesis fails, is panic the correct action?
```

Comment:
I think when `InitGenesis` failed , it is ok to panic , because the blockchain should not start anyway